src/main/java/org/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -202,7 +195,7 @@ protected Void call() {
             }
         };
         new Thread(task).start();
```

Comment:
DO we really need it to execute on a new thread or can't we reuse one of ours from the thread pools?

Reply:
Good observation. Fixed.